app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -658,4 +659,11 @@ def self.datacenter_inv_to_hashes(inv, cluster_uids, vm_uids, storage_uids, host
 
     result
   end
+
+  def self.vm_memory_reserve(vm_inv)
+		in_bytes = vm_inv.attributes.fetch_path(:memory_policy, :guaranteed)
+		in_bytes.nil? ? nil : in_bytes / Numeric::MEGABYTE
```

Comment:
I think you can just write
`vm_inv.attributes.fetch_path(:memory_policy, :guaranteed).to_i / Numeric::MEGABYTE`